### PR TITLE
tests: followup on testing against ansible2.4

### PR DESCRIPTION
**Pull request by guits**
ceph-ansible is now being testing against ansible2.2 and ansible2.4. We
need to update tox.ini so we use the right version of testinfra
regarding which ansible version we are using.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
jenkins this please

File: tox.ini [hunk 1]
```diff
@@ -154,9 +154,11 @@ setenv=
   bluestore_lvm_osds: CEPH_STABLE_RELEASE = luminous
 deps=
   ansible2.2: ansible==2.2.3
+  ansible2.2: -r{toxinidir}/tests/requirements2.2.txt
   ansible2.3: ansible==2.3.1
+  ansible2.3: -r{toxinidir}/tests/requirements2.2.txt
```
Comment (guits) [inline tox.ini]: not a typo. ansible2.2 and ansible2.3 require the same stack versions

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Pull request merged**